src/Root.cpp

```diff
@@ -607,7 +607,7 @@ bool cRoot::FindAndDoWithPlayer(const AString & a_PlayerName, cPlayerListCallbac
 			size_t Rating = RateCompareString (m_PlayerName, a_pPlayer->GetName());
 			if ((Rating > 0) && (Rating >= m_BestRating))
 			{
-				m_BestMatch = a_pPlayer;
+				m_BestMatch = a_pPlayer->GetName();
```

Comment:
I guess it's good enough as it is, this API is still pretty useless in my opinion, so what that it's half broken :P